packages/labs/eslint-plugin/src/test/index_test.ts

```diff
@@ -0,0 +1,17 @@
+/**
+ * @license
+ * Copyright 2023 Google LLC
+ * SPDX-License-Identifier: BSD-3-Clause
+ */
+
+import {suite} from 'uvu';
+// eslint-disable-next-line import/extensions
+import * as assert from 'uvu/assert';
```

Comment:
One for another time - this has always bugged me. The poor package exports uvu has. So you can't import the js file. One of its many problems. 
There's now an esm version of chai (prerelease). So maybe we move the monorepo to that in future?

Reply:
Possibly yeah! Since this one is a node only project, that could work well. I've used it a bunch before so could set that up